tools/build-lib/augmented-types/index.js

```diff
@@ -133,22 +133,22 @@ function propagateAugmentableTypes(libPath, logger) {
           continue;
         }
         const augTypesEnd =
-          publicApiFileSource.indexOf(DELIMITER_END) + DELIMITER_END.length + 1;
```

Comment:
Is removing the `+ 1`  intentional?

It changes the augTypesEnd value of augTypesEnd and then we make changes like `.substr(augTypesEnd);` to `.substring(augTypesEnd);`